From 080517ccf328b4798395bf01ef7ea9167f94a8e5 Mon Sep 17 00:00:00 2001
From: Jeffrey Mouroux <jeff.mouroux@xilinx.com>
Date: Wed, 26 Jul 2017 17:21:15 -0700
Subject: [PATCH 0191/1851] media: xilinx: Ensure Video Framebuffer selected by
 Kconfig

commit 3a7af7374b9dc8ddad463f9f5d42f51da0a2223a from
https://github.com/Xilinx/linux-xlnx.git

The Xilinx Video Linux driver is a client of the Video Framebuffer DMA
driver. Because the Framebuffer driver exports symbols used by this driver,
it is important that both are built in the same manner: as modules or
as statically included drivers.

Signed-off-by: Jeffrey Mouroux <jeff.mouroux@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/platform/xilinx/Kconfig b/drivers/media/platform/xilinx/Kconfig
index 7517e7046205..86f47c0ede6f 100644
--- a/drivers/media/platform/xilinx/Kconfig
+++ b/drivers/media/platform/xilinx/Kconfig
@@ -5,6 +5,7 @@ config VIDEO_XILINX
 	depends on VIDEO_V4L2 && VIDEO_V4L2_SUBDEV_API && OF && HAS_DMA
 	select VIDEOBUF2_DMA_CONTIG
 	select DMADEVICES
+	select XILINX_FRMBUF
 	select V4L2_FWNODE
 	help
 	  Driver for Xilinx Video IP Pipelines
-- 
2.31.1

